### ISSUE TITLE
Some transparent exports have solid frame background glitches.

### ISSUE THREAD

**Issue by FuzzyWobble** [jnordberg/gif.js#111]
![_3dtext_glitch](https://user-images.githubusercontent.com/1490568/67038424-5970f080-f0ed-11e9-9252-0b3f2d8c50ae.gif)

^ any idea what causes this glitch? happens about 10% of the time. 

Site is here: [https://3dtext2gif.com/](https://3dtext2gif.com/)

**Comment by FuzzyWobble** [comment]
*bump*

**Comment by rdsilver** [comment]
@FuzzyWobble did you ever figure this out?

**Comment by FuzzyWobble** [comment]
Nope. Seems to have something to do with the performance. Never fixed it.

**Comment by jrotty** [comment]
I also encountered this problem, but I changed the transparent value and it worked. But I am still worried that if the image material changes, it will still be triggered
